terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/el/ElTest.java

```diff
@@ -356,7 +356,7 @@ public void test06_Query_Display() {
         // output 06_05 Test
         assertThat(
                 driver.findElement(
-                        By.xpath("//a[contains(@href, \"?page=1&size=10&age=0&countries=&dateOfBirth=&main=false&name=%253Cscript%253Ealert('XSS%2520Attack')%253C/script%253E\")]"))
+                        By.xpath("//a[contains(@href, \"?page=1&size=10&age=0&_countries=&_dateOfBirth=&main=false&name=%3Cscript%3Ealert('XSS%20Attack')%3C/script%3E\")]"))
                         .getText(), is("2"));

```

Comment:
↑
I resolved bad implementation in this PR. Please refer to [diff](https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/235/files#diff-8f1e131ec73704a47760f93932c83567) of `beanQueryOutput.jsp`.